### ISSUE TITLE
[Feature Request] Delete entries with delete and backspace key

### ISSUE THREAD

**Issue by howdytom** [keeweb/keeweb#1906]
**Is your feature request related to a problem? Please describe.**

Deleting entries in the list view involves a lot mouse and click actions. 

**Describe the solution you'd like**

Deleting entries with the delete and backspace key would be a great time-saver. This feature request is kind of related to [#1900](https://github.com/keeweb/keeweb/issues/1900).

**Comment by antelle** [comment]
Deleting entries with a delete key can lead to unexpected effects, because deletion is kind of hard to revert in KeeWeb. Not sure how it should be properly implemented for now.

**Comment by howdytom** [comment]
Yes, totally got it. 

1. However we could show a warning before moving entries to the trash. 
2. Additionally the default button state for the warning dialog should be set **No**. 
 
Users have to explicitly select **Yes** in order to move entries to the trash. This will prevent bulk deletion. I have created a quick mockup.

![KeeWeb-Delete-warning dialog-using-delete-key](https://user-images.githubusercontent.com/9871520/134767715-ffb64407-76c8-4f7e-9570-5c40dd41ffd7.png)

**Comment by antelle** [comment]
Maybe if the deletion is done with the delete / backspace key, then we can show the dialog, you're right.

**Comment by antelle** [comment]
I'm not sure why the default state should be set to No though. If someone pressed Delete and then Enter, they for sure had an intention to delete the record.

**Comment by howdytom** [comment]
@antelle Good point. It could be an additional protection when someone presses too many times the Delete/Backspace key.

Going one step further: The same default state could be useful when you empty the trash. So you have to actively confirm deletion. On the other side it could be annoying too.

**Comment by LoganTann** [comment]
> However we could show a warning before moving entries to the trash
Is a warning really needed ?

Because the trash is here to prevent unexpected deletions

Anyway, I'd love to see such feature !!
